### DISCUSSION TITLE
It is possible to generate coverage including untested files?

### DISCUSSION THREAD

**Discussion by ciandt-crodrigues** [anishkny/playwright-test-coverage#132] [Q&A] [answered]
It is possible to generate coverage including untested files?
Today I run jest and playright for testing, and im using istanbul-merge to generate my coverage. I'd like to include the untested files so I can get a better report on what I need to test. Any suggestions?

**Comment by anishkny** [comment, marked as answer]
@ciandt-crodrigues sure you can try configuring `nyc` to [`include`](https://github.com/istanbuljs/nyc#common-configuration-options) untested files that you want.
